### ISSUE TITLE
Errors during Relese build

### ISSUE THREAD

**Issue by Tx707** [Fredi/IRCClient#5]
Using in Visual Studio 2013 whenever I try to compile I get the following errors:
http://pastebin.com/dVzi38Vg

Any news on how to compile it? It's a great source and I would like to use it.

**Comment by Fredi** [comment]
Hi! Did your VS 2013 converted the project when you first opened it?
I just compiled it in VS 2015 Professional and it worked.

**Comment by Tx707** [comment]
Hey. Sorry about the delay was absent from home. I've tried downloading it again and opening up the project file. It asked me if I wanted to upgrade and I clicked yes but it didn't generate a report(html file) like it normally does when you click yes. Tried the release compile and it would still give me the above errors. I have the community version of VS 2013. I'll try downloading the professional and I'll reply back.
